assets/js/app/Data.js

```diff
@@ -183,10 +183,12 @@ LocusZoom.Data.Source.prototype.getData = function(state, fields, outnames, tran
 };
 
 
-LocusZoom.Data.Source.prototype.parseResponse  = function(x, chain, fields, outnames, trans) {
-    var records = this.parseData(x.data || x, fields, outnames, trans);
-    var res = {header: chain.header || {}, body: records};
-    return res;
+LocusZoom.Data.Source.prototype.parseResponse = function(resp, chain, fields, outnames, trans) {
+    var json = resp;
+    if (typeof resp == "string"){ json = JSON.parse(resp); }
+    if (typeof json.data != "undefined"){ json = json.data; }
```

Comment:
@MrFlick - I tried to make this, being the default `parseResponse` method, as generic as possible. The response coming back could already be parse JSON, hence the type check conditional, and not all responses will present all the data under a top-level `data` attribute, hence the check for that. It feels a bit messy, so let me know what you think.